plugins/certs/subcommands/download

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eo pipefail
+[[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/certs/functions"
+
+cmd-certs-download() {
+  declare desc="exports SSL cert or key via STDOUT"
+  declare cmd="$1"
+  [[ "$1" == "$cmd" ]] && shift 1
+  declare APP="$1" KEY_TYPE="$2"
+
+  [[ -z "$APP" ]] && dokku_log_fail "Please specify an app to run the command on"
+  verify_app_name "$APP"
+  local APP_SSL_PATH="$DOKKU_ROOT/$APP/tls"
+
+  if [[ -f "$APP_SSL_PATH/server.key" ]] && [[ -f "$APP_SSL_PATH/server.crt" ]]; then
```

Comment:
Can we flatten this `if/else/fi` block? So exit early if the app doesn't have ssl.

Also, is there a helper function for checking for ssl support for an app?

Reply:
Updated.